src/main/groovy/jenkins/automation/utils/CommonUtils.groovy

```diff
@@ -0,0 +1,22 @@
+package jenkins.automation.utils
+
+/**
+ * Utils class when most reused common properties should live.
+ *  Adds a minimum base functionality required -build claiming, notifications and log.
+ * @param context  delegate passed in context\
+ */
+class CommonUtils {
+    static void add_defaults(context) {
+        context.with {
+            wrappers {
+                colorizeOutput()
+            }
+            logRotator {
+                numToKeep(10)
```

Comment:
I think 10 is too low. Let's change the default to something more reasonable and in line with how a lot of our jobs are currently configured.... I'd say at least 30